user_validator.py

```
@@ -0,0 +1,101 @@
+    if len(phone) != 10:
+        return False
+
+    return phone.isdigit()
+
+def execute_sql_query(user_id):
+    """
+    Execute SQL query - SECURITY RISK: SQL injection vulnerability.
+    This function directly embeds user input in SQL query.
+    """
+    # WARNING: This is vulnerable to SQL injection
+    query = f"SELECT * FROM users WHERE id = {user_id}"
```

Ruff (0.14.14)

Possible SQL injection vector through string-based query construction

(S608)